src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -343,6 +355,11 @@ const PopularRepositoriesTable = () => {
 
   const countIsZero = !data?.data?.length;
 
+  const clearFilters = () => {
```

Comment:
As this is just setting the searchValue to "". 
Do we need a function for this? or can we call it directly.

Reply:
So the function might be needed

Reply:
It's not. if you remove searchQuery/setSearchQuery and use searchValue/setSearchValue. 
Then replace this function "clearFilters" with ()=> setSearchValue('') 
You will have the same functionality, and you will actually be able to "search"